### PR TITLE
More SR methods

### PR DESCRIPTION
**Pull request by yosemitebandit**
PTAL --
- adds convenience methods on the SR, stuff like `add_number` and `get_port`
- makes integration test more like a test

### CODE REVIEW

File: openbts/components.py [hunk 1]
```diff
@@ -50,50 +50,130 @@ def __init__(self, address='tcp://127.0.0.1:45064'):
...
+    """Gets subscribers, optionally filtering by IMSI.
 
     Args:
       imsi: the IMSI to search by
-      name: the Name to search by
-      msisdn: the number to search by
 
     Returns:
-      Response instance
+      array of subscriber dicts
```
Comment (shaddi) [inline openbts/components.py]: Could you list what the dict contains or even an example return value?

File: openbts/components.py [hunk 2]
```diff
@@ -133,12 +213,39 @@ def delete_subscriber(self, imsi):
...
+      'match': {
+        'name': imsi
+       },
+      'fields': {
+        'port': new_port,
+       }
+    }
+    return self._send_and_receive(message)
+
+  def update_subscriber(self, imsi, new_msisdn, new_ipaddr, new_port):
```
Comment (shaddi) [inline openbts/components.py]: Perhaps we could make these default to `None` and only update what's needed? This is assuming it's possible to do this update w/o requiring a lookup first.

**Comment by shaddi** [comment]
couple small suggestions but lgtm overall

File: openbts/components.py (hunk 1 above)
Comment (yosemitebandit) [inline openbts/components.py]: good point! done

File: openbts/components.py (hunk 2 above)
Comment (yosemitebandit) [inline openbts/components.py]: this method is unused now that we have all these tiny setters, so I'm just removing it entirely.

**Comment by yosemitebandit** [comment]
fixes #3

**Pull request merged**